lms/djangoapps/badges/tests/factories.py

```diff
@@ -0,0 +1,81 @@
+"""
+Factories for Badge tests
+"""
+from random import random
+
+import factory
+from django.core.files.base import ContentFile
+from factory import DjangoModelFactory
+from factory.django import ImageField
+
+from badges.models import BadgeAssertion, CourseCompleteImageConfiguration, BadgeClass, CourseEventBadgesConfiguration
+from student.tests.factories import UserFactory
+
+
+def generate_dummy_image(_unused):
+    """
+    Used for image fields to create a sane default.
+    """
+    return ContentFile(
+        ImageField()._make_data(  # pylint: disable=protected-access
+            {'color': 'blue', 'width': 50, 'height': 50, 'format': 'PNG'}
+        ), 'test.png'
+    )
+
+
+class CourseCompleteImageConfigurationFactory(DjangoModelFactory):
+    """
+    Factory for BadgeImageConfigurations
+    """
+    class Meta(object):
+        model = CourseCompleteImageConfiguration
+
+    mode = 'honor'
```

Comment:
I'd rather not because I'd probably have to change some of the certificate tests, too, which still use 'honor'.